### ISSUE TITLE
Open a menu in the application menu bar

### ISSUE THREAD

**Issue by ebai101** [Hammerspoon/hammerspoon#2420]
I'm trying to bind a hotkey to open the File menu. I'm coming from Keyboard Maestro, where there's a "Select or Show Menu Item" function that can just display a menu, and from there I can start typing the name of a menu item to select it. It doesn't seem like there's a way to do this in vanilla Hammerspoon, although I could be wrong - maybe there's an osascript-based solution?

**Comment by latenitefilms** [comment]
Does this do what you want?

[hs.application:selectMenuItem(menuitem[, isRegex])](http://www.hammerspoon.org/docs/hs.application.html#selectMenuItem)

**Comment by ebai101** [comment]
No, that doesn't work unfortunately. Here's an example - I want the code below to reveal the File menu in the frontmost app when I press F12:

```
hs.hotkey.bind({}, 'f12', function()
	hs.alert.show('showing the file menu')
	hs.application.frontmostApplication():selectMenuItem({'File'})
end)
```

I think this is the same quirk that I ran into with JXA (see [here](https://github.com/JXA-Cookbook/JXA-Cookbook/wiki/System-Events#clicking-menu-items)) where a simulated click on an item with no action doesn't actually do anything.

Let me know if I'm missing something. I'm on High Sierra 10.13.6 for what it's worth.

**Comment by latenitefilms** [comment]
Ummm, that's strange - you're code is working fine for me with Safari, but I'm on Mojave 10.14.6.

**Comment by latenitefilms** [comment]
As a workaround, you could also use [`hs._asm.axuielement`](https://github.com/asmagill/hs._asm.axuielement) instead.
